packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,95 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
```

Comment:
If there are no variables, you should remove this import and the empty mixin file.